aldryn_grid_foundation/models.py

```diff
@@ -13,7 +14,7 @@
 
 
 ALDRYN_GRID_FOUNDATION_CHOICES = [
-    (i, string_concat(str(i), ' ', _('columns'))) for i in range(1, GRID_CONFIG['COLUMNS']+1)
+    (i, format_lazy(str(i), ' ', _('columns'))) for i in range(1, GRID_CONFIG['COLUMNS']+1)
```

Comment:
Could you please add these lines into the top of the file.
A let `string_concat` be on the same place.

```
try:
    from django.utils.translation import string_concat
except ImportError:
    from django.utils.text import format_lazy

    def string_concat(*strings):
        return format_lazy('{}' * len(strings), *strings)
```